service/src/main/kotlin/app/cash/backfila/service/runner/BackfillRunnerLoggingSetupProvider.kt

```diff
@@ -0,0 +1,48 @@
+package app.cash.backfila.service.runner
+
+import app.cash.backfila.service.persistence.DbBackfillRun
+import misk.hibernate.Id
+import misk.logging.getLogger
+import org.slf4j.MDC
+
+interface BackfillRunnerLoggingSetupProvider {
```

Comment:
Maybe there is a better name I'm not thinking of.

Reply:
broader question, can the client and server have the same interface? consistent MDC for both?

Reply:
no harm in having 2 just in case theres some diffs I guess

Reply:
Yeah that was my thoughts as well.